### ISSUE TITLE
AutomationPeer returns wrong value for "enabled" attribute

### ISSUE THREAD

**Issue by nrymanov** [AvaloniaUI/Avalonia#11370]
**Describe the bug**
For buttons bound to commands, Automation Peers returns an incorrect "enabled" attribute value if the command is disabled (CanExecute() returns false).

**To Reproduce**

1. Install WindowsApplicationDriver
2. Download and build the attached solution [AutomationTest.zip](https://github.com/AvaloniaUI/Avalonia/files/11472971/AutomationTest.zip)
3. Run Tests

**Expected behavior**
[If a button is disabled due to the command cannot be executed at the time, the corresponding AutomationPeer should returns "False" as the value of the "enabled" attribute.]

**Desktop (please complete the following information):**

- OS: Windows, not tested on Mac and Linux
- Version [11.0.0-preview8]
